src/pages/blogs-page/blogs-page.tsx

```diff
@@ -0,0 +1,82 @@
+import { SimpleGrid } from '@chakra-ui/icons';
+import { Heading, VStack } from '@chakra-ui/react';
+import { useEffect, useState } from 'react';
+import { useNavigate } from 'react-router';
+
+import { PATHS } from '~/app/routes/paths.ts';
+import { BlogCard, NewestRecipes, SectionWrapper } from '~/components';
+import { OtherBlogsCollapse } from '~/components/widgets/other-blogs-collapse/other-blogs-collapse.tsx';
+import { BLOGS_FAV_BOX, BLOGS_FAV_GRID } from '~/constants/data-test-ids.ts';
+import { TOAST_MESSAGES } from '~/constants/toast-messages.ts';
+import { useCustomToast } from '~/hooks/use-custom-toast.tsx';
+import { useScreenSize } from '~/hooks/use-screen-size.tsx';
+import { useGetBlogsQuery } from '~/query/services/blogs/blogs-api.ts';
+import { useAppSelector } from '~/redux/hooks.ts';
+import { selectUserId } from '~/redux/slices/auth-slice.ts';
+
+const { ServerErrorToast } = TOAST_MESSAGES;
+
+export const BlogsPage = () => {
+    const { isMidDesktop } = useScreenSize();
+    const [blogsPerView, setBlogsPerView] = useState('9');
+    const { toast } = useCustomToast();
+    const navigate = useNavigate();
+    const currentUserId = useAppSelector(selectUserId);
+    const { data: blogsPreview, isError } = useGetBlogsQuery({
+        currentUserId,
+        limit: blogsPerView,
```

Comment:
если всем эти игры с числами-строками только из за того, что тут должна приходить строка, то может быть сделаем вот так?
```suggestion
        limit: String(blogsPerView),
```

Reply:
а дальше по файлу будем использовать числа. Которые, кстати, стоит вынести в константы

Reply:
выше по файликам видел тоже эти числа-строки. Если логику не сломает, то лучше поправить